pep-0710.rst

```diff
@@ -118,9 +118,9 @@ may be present in a given ``.dist-info`` directory; installers MUST NOT add both
 The ``provenance_url.json`` JSON file MUST be a dictionary, compliant with
 :rfc:`8259` and UTF-8 encoded.
 
-If present, it MUST contain exactly two keys. The first one is ``url``, with
+If present, it MUST contain exactly three keys. The first one is ``url``, with
 type ``string``.  The second key MUST be ``archive_info`` with a value defined
```

Comment:
For consistency with the second and third keys (suggestion split in two parts due to GH interface)

```suggestion
If present, it MUST contain exactly three keys. The first one MUST be ``url``,
with type ``string``.  The second key MUST be ``archive_info`` with a value
```

Reply:
Adjusted, thanks!